CredentialManager/app/src/main/java/com/google/credentialmanager/sample/Auth.kt

```diff
@@ -54,8 +61,13 @@ class Auth(context: Context) {
         activity: Activity,
         creationResult: JSONObject
     ): GetCredentialResponse? {
-        Toast.makeText(activity, "Fetching previously stored credentials", Toast.LENGTH_SHORT)
-            .show()
+
+        if(!isPasskeySupported(activity)) {
+            Toast.makeText(activity, "Passkeys are not supported on this devices", Toast.LENGTH_SHORT)
```

Comment:
Update to make the last word singular: "Passkeys are not supported on this device."

Reply:
Decided not to show any UI elements since it's a business logic and should be decoupled from UI . Added just logs.
We can show some UX based on state change and address it in further improvements.